test/support/conn_case.ex

```diff
@@ -74,6 +74,19 @@ defmodule ScreenplayWeb.ConnCase do
 
           {conn, user}
 
+        tags[:api_unauthenticated] ->
```

Comment:
Being unauthenticated is sort of the default state — do we need a tag for this? I think in the couple of tests where we specifically test providing an _incorrect_ API key, it would actually make things clearer to put that logic in the tests themselves. (It seems like we'll want the `authenticated` tag though, because we'll have many tests that require an authenticated connection but aren't really intended to test the authentication itself.)

Reply:
Good point. Went a little tag happy.